express/blocks/template-list/template-list.js

```diff
@@ -1859,36 +1877,64 @@ function addBackgroundAnimation($block, animationUrl) {
-      $block.innerHTML = $block.innerHTML.replaceAll('default-format', relevantRowsData.placeholderFormat || '');
-
-      if (relevantRowsData.templateTasks === '') {
-        $block.innerHTML = $block.innerHTML.replaceAll('default-create-link-text', placeholders['start-from-scratch'] || '');
-      } else {
-        $block.innerHTML = $block.innerHTML.replaceAll('default-create-link-text', relevantRowsData.createText || '');
+async function replaceRRTemplateList($block) {
+  const placeholders = await fetchPlaceholders().then((result) => result);
+  const relevantRowsData = await fetchRelevantRows(window.location.pathname);
+  props.limit = parseInt(placeholders['relevant-rows-templates-limit'], 10) || 10;
+
+  if (relevantRowsData) {
+    $block.closest('.section').dataset.audience = 'mobile';
+    props.headingTitle = relevantRowsData.header || null;
+    props.headingSlug = relevantRowsData.shortTitle || null;
+    props.viewAllLink = relevantRowsData.viewAllLink || null;
+
+    if (relevantRowsData.manualTemplates === 'Y') {
+      const $sectionFromFragment = await fetchPlainBlockFromFragment(`/express/fragments/relevant-rows/${relevantRowsData.templateFragment}`, 'template-list');
+      const $newBlock = $sectionFromFragment.querySelector('.template-list');
+
+      if ($newBlock) {
+        const $section = $block.closest('.section');
+        const $sectionHeading = $section.querySelector('div.default-content-wrapper > h2');
+        let $sectionSlug = null;
+
+        if ($sectionHeading.textContent.trim().indexOf('{{heading_placeholder}}') >= 0) {
+          if ($block.classList.contains('spreadsheet-powered') && props.headingTitle) {
+            $sectionHeading.textContent = props.headingTitle || '';
+
+            if (props.headingSlug) {
+              $sectionSlug = createTag('p');
+              $sectionSlug.textContent = props.headingSlug;
+            }
+          }
+        }
+        $block.classList.remove('apipowered');
+        $block.innerHTML = $newBlock.innerHTML;
+        await fixIcons($block);
       }
+    }
+
+    $block.innerHTML = $block.innerHTML.replaceAll('default-title', relevantRowsData.shortTitle || '');
+    $block.innerHTML = $block.innerHTML.replaceAll('default-tasks', relevantRowsData.templateTasks || '');
+    $block.innerHTML = $block.innerHTML.replaceAll('default-topics', relevantRowsData.templateTopics || '');
+    $block.innerHTML = $block.innerHTML.replaceAll('default-locale', relevantRowsData.templateLocale || 'en');
+    $block.innerHTML = $block.innerHTML.replaceAll('default-premium', relevantRowsData.templatePremium || '');
+    $block.innerHTML = $block.innerHTML.replaceAll('default-animated', relevantRowsData.templateAnimated || '');
+    $block.innerHTML = $block.innerHTML.replaceAll('https://www.adobe.com/express/templates/default-create-link', relevantRowsData.createLink || '/');
```

Comment:
Can there be localized links like https://www.adobe.com/de/express/templates/default-create-link ?